README.md

```diff
@@ -70,12 +73,13 @@ class Person {
 
 ### 3. Create a DAO (Data Access Object)
 
-This component is responsible for managing access to the underlying SQLite database.
-The abstract class contains the method signatures for querying the database which have to return a `Future` or `Stream`.
+This component is responsible for managing access to the underlying SQLite database. The abstract
+class contains the method signatures for querying the database which have to return a `Future`
+or `Stream`.
 
-- You can define queries by adding the `@Query` annotation to a method.
-  The SQL statement has to get added in parenthesis.
-  The method must return a `Future` or `Stream` of the `Entity` you're querying for.
+- You can define queries by adding the `@Query` annotation to a method. The SQL statement has to get
```

Comment:
It might be nice to rewrite this just a little bit: 

```
You can define queries by adding the `@Query` annotation to a method. The SQL statement then has to 
be added in parenthesis. The method must return a `Future` or `Stream` with the primitive
types `String`, `int`, `double`, `bool`, `Uint8List`, or a Floor `Entity` you're querying for.
```

Reply:
Done :)